resources/views/components/article-listing.blade.php

```diff
@@ -4,7 +4,7 @@
     $url => string '/news/'
     $link_text => string // 'More news'
 --}}
-<h2>{{ $heading ?? 'News' }}</h2>
+<h2{!! !empty($class) ? ' class="'.$class.'"' : '' !!}>{{ $heading ?? 'News' }}</h2>
```

Comment:
Can we add this to the documentation on the top of the file for each of these?

Reply:
yep! https://github.com/waynestate/base-site/pull/306